back/engines/commercial/report_builder/app/services/report_builder/queries/survey_results.rb

```diff
@@ -0,0 +1,8 @@
+module ReportBuilder
+  class Queries::SurveyResults
+    def run_query(project_id:, **_other_props)
```

Comment:
Since James removed continuous projects, should we stop accepting the `project_id` parameter here and only accept the `phase_id` instead?

Reply:
Very good point. I fixed it like that https://github.com/CitizenLabDotCo/citizenlab/pull/6605/commits/c722e82d7c43a037baa67887412b32b7bd61cbcc